operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -208,11 +206,6 @@ data:
           - kube_pod_container_resource_limits 
           - kube_pod_container_resource_requests 
           - kube_resourcequota 
-          - mixin_pod_workload 
-          - namespace_workload_pod:kube_pod_owner:relabel 
-          - node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate 
-          - node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate 
-
```

Comment:
Removed recording rules from the collection rules list. These will still be captured within the default metrics list.